portal/src/graphql/adminapi/mutations/deleteIdentityMutation.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback } from "react";
+import { useMutation, gql } from "@apollo/client";
+import { DeleteIdentityMutation } from "./__generated__/DeleteIdentityMutation";
+
+const deleteIdentityMutation = gql`
+  mutation DeleteIdentityMutation($identityID: ID!) {
+    deleteIdentity(input: { identityID: $identityID }) {
+      success
+    }
+  }
+`;
+
+export function useDeleteIdentityMutation(): {
+  deleteIdentity: (identityID: string) => Promise<boolean>;
+  loading: boolean;
+  error: unknown;
+} {
+  const [mutationFunction, { error, loading }] = useMutation<
+    DeleteIdentityMutation
+  >(deleteIdentityMutation, { refetchQueries: ["UserDetailsScreenQuery"] });
```

Comment:
I suppose we may want to extract the query to an individual file, to avoid referencing screen-specific queries in a generic mutation.